### PR TITLE
Add a note about satellite:el8 where appropriate

### PR DESCRIPTION
**Pull request by adamlazik1**
Made the note about satellite:el8 into a snippet and included where appropriate.
The second commit is a response to [BZ#2141133](https://bugzilla.redhat.com/show_bug.cgi?id=2141133)

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7
* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

**Review by maximiliankolb — COMMENTED** [review]
Changes look good to me.

File: guides/common/modules/proc_preparing-a-host-for-external-databases.adoc [hunk 1]
```diff
@@ -41,13 +41,15 @@ endif::[]
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-ifdef::satellite[]
-# dnf module enable satellite:el8
-endif::[]
-ifdef::katello[]
-# dnf module enable katello:el8
-endif::[]
+# dnf module enable {dnf-modules}
```
[the remark this answers is not in the document]
Comment (ekohl) [reply guides/common/modules/proc_preparing-a-host-for-external-databases.adoc]: I see you already fixed it, but for background: `pulpcore:el8` is actually the module that contains `postgresql-evr`. Leaving it out was a bug. Then `foreman:el8` (pulled in by `katello:el8`) ensures PostgreSQL 12 is used so `{dnf-modules}` is correct.

**Review by ekohl — APPROVED** [review]

**Comment by adamlazik1** [comment]
Just realized I forgot end end the snippet with `.adoc` suffix to the snippet. That being the only change I made, I will proceed with merging once the checks complete since the PR has been approved.

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   1ade5daae..808292645  3.4 -> 3.4
   9c0489603..1c1a0f199  3.5 -> 3.5
I realized there is no {dnf-modules} attribute on branches 3.3 and lower, so I will make a separate PR against 3.3 that can be cherry-picked to 3.2 and 3.1.

**Pull request merged**